examples/simple-generator/tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+    "include": ["src/**/*.ts"],
+    "compilerOptions": {
+        "strict": false,
```

Comment:
Is `"strict": true` a better option?

Reply:
Changed. https://github.com/SAP/open-ux-tools/pull/466/commits/2d2c2a4316c8a4eca839b5f0ccefc4dc13f44de8